package.json

```diff
@@ -12,11 +12,11 @@
     "lint:watch": "watch 'npm run lint:fix' . --wait=1",
     "test": "npm run lint && npm run flow",
     "bpf-sdk:update": "node_modules/@solana/web3.js/bin/bpf-sdk-install.sh",
-    "build": "webpack --mode=production",
+    "build": "",
```

Comment:
why remove this?

Reply:
There is not webapp in this project
`ERROR in Entry module not found: Error: Can't resolve './src/webapp/index.js'`

There is probably more/better cleanup to do related to the removed webapp, suggestions?

Reply:
Ah, right.   I'd just remove the build script entirely, and `builddev`/`dev`, delete `webpack.config.js`, and the webpack dependencies in package.json.  That's a good enough scrub